Upgrading DB cersion causes users with previous DB version to crash on launch

Upgrading the `TxDatabase` through : `@Database(entities = [Tx::class], version = 3, exportSchema = false)` will cause users with previous DB versions, for example version 2 to crash on launch with the following : 
```
java.lang.RuntimeException: Exception while computing database live data.
        at androidx.room.RoomTrackingLiveData$1.run(RoomTrackingLiveData.java:92)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:920)
     Caused by: java.lang.IllegalStateException: A migration from 2 to 3 was required but not found. Please provide the necessary Migration path via RoomDatabase.Builder.addMigration(Migration ...) or allow for destructive migrations via one of the RoomDatabase.Builder.fallbackToDestructiveMigration* methods.
```
We can either clear the DB every time we upgrade it or add a method to migrate their DBs to the correct version

--- a/app/src/main/java/com/goldenraven/padawanwallet/data/TxDatabase.kt
+++ b/app/src/main/java/com/goldenraven/padawanwallet/data/TxDatabase.kt
@@ -29,7 +29,9 @@ abstract class TxDatabase: RoomDatabase() {
                     context.applicationContext,
                     TxDatabase::class.java,
                     "transaction_history",
-                ).build()
+                )
+                    .fallbackToDestructiveMigration()
+                    .build()
                 INSTANCE = instance
                 return instance
             }

